readme.md

```diff
@@ -129,3 +129,12 @@ mean(mtcars$mpg)
 plot(mtcars$mpg, mtcars$wt)
 ```
 ````
+
+## Missing features of Jupyter notebook
+
+Jupyter notebooks are rendered into HTML with quarto using the Jupyter kernel. Comparing to native quarto documents, the following features are not supported.
+
+* Built-in YAML Metadata
```

Comment:
The YAML header must be in the first cell that must be of type `raw`. See https://quarto.org/docs/tools/jupyter-lab.html#yaml-front-matter.

Reply:
Thank you very much! It should been fixed.